src/_standard.scss

```diff
@@ -4,19 +4,19 @@
 ////
 
 @mixin oQuoteStandardIcon {
-	position: absolute;
-	top: oTypographySpacingSize($units: 4);
-	left: oTypographySpacingSize($units: 2);
-	@include oIconsGetIcon('speech-left', oColorsGetPaletteColor('claret-80'), 50, $iconset-version: 1);
+	@warn "oQuoteStandardIcon is now deprecated and will be removed in the next major release";
```

Comment:
I think removing styles from a mixin is a breaking change

Reply:
@JakeChampion we discussed with Arjun and decided this was fine. The icon shouldn't display any more, and this way it will be hidden even if the implementation uses the deprecated markup.